src/components/card/card.tsx

```diff
@@ -27,21 +27,22 @@ const Card = ({ card, stats, turn, cardNum, active }: Props) => {
     const img = cardTypeIcons[card.type]
     const isActive = active === cardNum
     const disabled = (!isActive && active) || card.cost > stats[resMap[card.type]]
+    const cardBack = (turn === 2 && !isActive)
 
     return (
         <>
-            <div id={`card-${cardNum}`} className={`${styles[`${card.type}`]} ${styles.cardBody} ${(!disabled && !isActive && turn !== 2) && styles.hoverEffect}`}>
-                {(turn === 2 && !isActive) && <div className='card-back'></div>}
-                <div hidden={!disabled || isActive} className={styles.disabled}></div>
-                <span className={styles.titleSpan}>{card.name}</span>
-                <div className={styles.imgContainerDiv}>
-                    <img className={styles.img} src={imageFilename(card.name)} />
-                </div>
-                <span className={styles.textSpan}>{card.description}</span>
-                <span className={styles.cardCost}>{card.cost}</span>
-                <div className={styles.cardIcon}>
-                    <Image width={20} height={20} alt={`${card.type} icon`} src={img} />
-                </div>
+            <div id={`card-${cardNum}`} className={`${styles[`${card.type}`]} ${styles.cardBody} ${disabled && turn === 1 && !isActive && styles.disabled} ${(!disabled && !isActive && turn !== 2) && styles.hoverEffect} ${cardBack && 'card-back'}`}>
```

Comment:
I have added two additional classes conditionally. This increases the line length. We can come up with a more elegant way of handling the conditional application of many classes. But I intentionally excluded such a way in order to make this changeset a bit smaller.